crates/swift-bridge-build/src/package.rs

```diff
@@ -328,12 +350,46 @@ let package = Package(
 		),
 		.target(
 			name: "{package_name}",
-			dependencies: ["RustXcframework"])
+			dependencies: ["RustXcframework"],
+			resources: [
```

Comment:
Can we move the package string generation out into its own function https://github.com/chinedufn/swift-bridge/blob/03de7ea4fb646de1b71358a717488ddac5d67e97/crates/swift-bridge-build/src/package.rs#L313-L335

Then add a test to the bottom of this file that generates the package description for the `CreatePackageConfig` that has resources and asserts on the contents of the generated string?

My main concern here is that right now the writing to disk / process spawning stuff is tightly coupled to things that it doesn't need to be (I know this was here before you, so not saying that you did that!).

So I'm asking to add a test for this resources thing so that we at least begin take a small step in the right direction and make it easier to make our Swift package creation more powerful over time.

Reply:
I did make an effort to follow your (current) style of programming, imitating how things are done. I think it is better if you, as the owner of this project, set the new precedence instead of me as I will probably not do it as you like it.

Reply:
Actually all of the package stuff was contributed by a contributor.

However, makes sense to me!